src/scale.ts

```diff
@@ -259,10 +262,9 @@ export interface Scale {
   useRawDomain?: boolean;
 }
 
-export const SCALE_PROPERTIES = [
+export const SCALE_PROPERTIES:(keyof Scale)[]= [
   'type', 'domain', 'range', 'round', 'rangeStep', 'scheme', 'padding', 'clamp', 'nice',
-  'exponent', 'zero',
-  // TODO: add interpolate here
+  'exponent', 'zero', 'interpolate',
```

Comment:
Order this just like how we order Scale:

`['type', 'domain', 'range', 'rangeStep', 'scheme', ... (the rest of properties order alphabetically...]`